src/asciidoc/docs/develop/editor.adoc

```diff
@@ -36,30 +36,46 @@ For in-depth walkthroughs of working outside the Koji editor, see <<use-git#>>.
 
 == Previews
 
-=== Inline preview
+You can select the type of preview to display in the preview pane.
+See <<testing-templates.adoc#_using_previews,using previews>>.
```

Comment:
```suggestion
To learn more about using previews, see <<testing-templates#_testing_in_the_koji_editor,Testing in the Koji editor>>.
```
A few problems this suggestion is aiming to address:
* Adding more context so the reader knows what to expect before following the link.
* Fixing Asciidoc markup error (we don't use the `.adoc` in cross-references.